Modules/Loadable/Markups/MRML/vtkMRMLMarkupsROINode.h

```diff
@@ -129,7 +129,7 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsROINode : public vtkM
   //@}
 
   /// 4x4 matrix defining the object center and axis directions within the world coordinate system.
-  /// Changes made to the matrix will not be applied.
+  /// Users should avoid modifying this matrix.
   vtkMatrix4x4* GetObjectToWorldMatrix()
```

Comment:
This follows the suggestion of @lassoan on the [forum](https://discourse.slicer.org/t/unexpected-interactions-of-roi-objects-with-code/40053/3). However, it raises the question -- if (external) users are not to modify this matrix, why is the return type not `const vtkMatrix4x4*` (and then, why not make the method itself `const`)?

Reply:
VTK is not const correct, so you cannot just add `const` wherever you want, and it would not help anyway when the API is used from Python.